pkg/spec/backup.go

```diff
@@ -93,6 +93,9 @@ type S3Source struct {
 	// S3Bucket overwrites the default etcd operator wide bucket.
 	S3Bucket string `json:"s3Bucket,omitempty"`
 
+	// S3Path to be used to prefix the bucket path
```

Comment:
S3Path to be used to prefix the bucket path. It's the prefix at the beginning. After that, it will have version and cluster specific prefix appended.